core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -126,7 +127,7 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
             },
             isEnterpriseLicense: state.isEnterpriseLicense,
             state: state.editorState ?? EDITOR_STATE.LOADING,
-            previewState: state.previewState,
+            editorData: state.editorData,
```

Comment:
Not sure about this property and name... looks like is too general or generic, not sure.

Reply:
Im not sure too I had change this name like 3 times since I started working on this.